flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/hive/TestHivePropertiesConverter.java

```diff
@@ -35,7 +35,7 @@ public void testToGravitinoCatalogProperties() {
     Configuration configuration =
         Configuration.fromMap(
             ImmutableMap.of(
-                "hive-conf-dir",
```

Comment:
better to pass `hive-conf-dir` not `flink.bypass.hive-conf-dir` when creating a hive catalog from Flink SQL

Reply:
revert